src/coreclr/vm/appdomain.hpp

```diff
@@ -46,10 +46,10 @@ class DomainAssembly;
 class LoadLevelLimiter;
 class TypeEquivalenceHashTable;
 
-#ifdef FEATURE_COMINTEROP
+#if defined(FEATURE_COMINTEROP) || defined(FEATURE_COMWRAPPERS)
 class RCWCache;
```

Comment:
The `RCWCache` shouldn't be needed for the `ComWrappers` API, only the `RCWRefCache`.

Reply:
Ok tried to include RCWCache only for FEATURE_COMINTEROP and worked